### ISSUE TITLE
Simple feature. Pallet loading run

### ISSUE THREAD

**Issue by Frag2000** [Stephan-S/FS22_AutoDrive#898]
I am a long time user of Autodrive in FS19. Loves everything about it. I reinstalled it for FS22 lately. Still awesome. Great work!

But to my surprise, there is a simple thing that cannot be done. I would like my tractor to do a morning pallet run, meaning that it would go around my farm, and stop at many production points to pickup pallets with an Autoload trailer.

I RECORDED successfully the ole route, pausing here and there to let the pallet autoload. But I quickly noticed, while PLAYING the route, that the pause I did while recording the path is not kept. Briefly the tractor goes around the route smoothly, but will not pause where I did when I recorded it.

An easy way to provide that feature would be to simply add a "Pause interval" property, with an integer value in seconds that you could manually set while editing a waypoint. It could be in the UI editor, where you edit the name of a waypoint.

That little feature would let people program routes that pause here and there to let pallet load.

Short discussion on the FS22 forum ...

[https://forum.giants-software.com/viewtopic.php?p=1570148#p1570148](url)

Again, great work and thank you for such an interesting mod.

**Comment by QQQPAWEL** [comment]
adding two types of filling to the seed drill

**Comment by Frag2000** [comment]
Sorry but not sure what it does mean? The seed drill?

**Comment by Frag2000** [comment]
> In the AD settings you can allready set a timer, how long the vehicle should pause/wait if AutoLoad is used. In the menu entry with the tractory symbol scroll down to the bottom.

I found the property. However I am not sure how it can be done. I was not able to use it. 

I do have a circuit with 8 GREEN HOUSES. I did record a path in DRIVE mode where the truck passes close to all the loading zone. Then I played it back (with autoload on), thinking that the tractor would stop every time that the autoload start. But it did not. Same effect, the tractor drive around without stopping.

Briefly what are exactly the conditions to trigger the tractor to stop to respect that property pause interval? I thought that Autodrive would detect that pallet started to load on the trailer. But it seems not. I am probably missing something.

Any hints?
